### ISSUE TITLE
validation errors for annotation displays

### ISSUE THREAD

**Issue by martinschorb** [mobie/mobie-utils-python#64]
Hi,

I am getting validation errors for my views that contain annotation displays (Figure2x). (newest main commit v. 2.6)

```
File "~/code/mobie-utils-python/mobie/validation/utils.py", line 64, in _assert_true
    raise ValueError(msg)
ValueError: 'imageDisplay' is a required property
Failed validating 'required' in schema[0]:
    {'additionalProperties': False,
     'description': '',
     'properties': {'imageDisplay': {'additionalProperties': False,
                                     'description': 'Viewer state for a '
                                                    'group of image '
                                                    'sources.',

                                    ...

     'required': ['imageDisplay'],
     'type': 'object'}
On instance:
    {'sourceAnnotationDisplay': {'boundaryThickness': 0.2,
                                 'colorByColumn': 'annotationColor',
                                 'lut': 'argbColumn',
                                 'name': 'hm-tomo-annotations',
                                 'opacity': 1.0,
                                 'showAsBoundaries': True,
                                 'sources': {'1': ['tomo_38_hm']},
                                 'tableData': {'tsv': {'relativePath': 'tables/highmag_tomos'}},
                                 'tables': ['default.tsv']}}
```

this does not make sense. `sourceAnnotationDisplay`s exist in parallel to `imageDisplay`s don't they?

**Comment by constantinpape** [comment]
This is due to the spec updates. It will work again tomorrow when everything is synced, I will ping you then.

**Comment by constantinpape** [comment]
@martinschorb I have just merged #63 which should fix this. So if you pull the latest master everything should work again.
